dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/AuthorRole.cs

```diff
@@ -31,11 +31,6 @@ namespace Microsoft.SemanticKernel.AI.ChatCompletion;
     /// </summary>
     public static AuthorRole Tool { get; } = new("tool");
```

Comment:
Same goes for all of these, but it's also ok to leave them as they are. Just FYI.